taxi/urls.py

```diff
@@ -42,7 +46,24 @@
     path("cars/create/", CarCreateView.as_view(), name="car-create"),
     path("cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"),
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
+    path(
+        "cars/<int:pk>/assign-unassign/",
```

Comment:
choose a better name for endpoint, since `assign-unassign` is a bit too overloaded

Reply:
Fixed